### ISSUE TITLE
try catch for bad images

### ISSUE THREAD

**Issue by 123alaa** [AlexeyAB/darknet#2629]
Hi @AlexeyAB ,
Is there a simple to add try catch when the loaded image is bad (canot load with opencv)?
i found that you load the image in load_image_resize in http_stream.cpp file,
if i add try catch in the loading is is enough?
It is a problem, sometimes you have 1 from 20000 image that is bad and after 3 days of training the code code fail.

**Comment by aimhabo** [comment]
I think adding a boolean array to record which images in the current epoch have been loaded, you can guarantee that all images are loaded and loaded only once in each epoch. This only requires an epoch to report an error.
And if you catch a bad image, the *data of that image must to be null.

**Comment by AyushP123** [comment]
@123alaa, what do you mean by bad images here, images which opencv can't load ?? do you want to continue loading all the images despite a few being null ??

**Comment by 123alaa** [comment]
@AyushP123 , yes i want to continue loading the other images, and dont get segmentation fault.

**Comment by aimhabo** [comment]
@123alaa you could make_random_image() with an empty .txt replace the image that should have been loaded

**Comment by 123alaa** [comment]
i solve it, with iteration on all the images and try to load and remove the bad, but i excepted that it was implemented in the code.
